index.rst

```diff
@@ -6,65 +6,110 @@
 4. The **Electrometer** and **Fiber Spectrograph** are used to monitor the output beam. 
 5. The **Electronics Cabinet** provides power and control to the other hardware. 
 
 These elements are integrated on a large, heavy table. The White light source, monochromator, and optics are all housed in a light tight box. 
 
+.. figure:: /_static/full_lab.png
+ :name: full_lab
+ :target: ../_images/full_lab.png
+ :alt: full_lab
+
+ AuxTel Calibration Illumination System setup in the Tucson Lab.
+
+.. figure:: /_static/left_door.png
+ :name: left_door
+ :target: ../_images/left_door.png
+ :alt: left_door
+ :scale: 50 %
+
+ Left side of Illumination as seen in the AT dome.
+
+.. figure:: /_static/right_door.png
+ :name: right_door
+ :target: ../_images/right_door.png
+ :alt: right_door
+ :scale: 50 %
+
+ Right side of Illumination as seen in the AT dome.
+
 White Light Source
 ------------------
 The white light source is a KiloArc Lamp built by Optical Building Blocks. It can output white light with a power of 800 - 1200 W. This light is directed into the monochromator.
 
-Because of the heat generated by such a lamp, before sending the light into the monochromator, the light travels through a water source which absorbs much of the infrared wavelengths. This water heats up, so the white light source must be operated with a chiller, discussed below. The chiller should keep the water cool enough, but if there is ever a situation where there is too much heat being sent into the monochromator, there is a thermal fuse glued attached to the entrance to the monochromator. When it hits a certain temperature it will trigger the emergency stop. On the mountain, there are also temperatures that will freeze the water in the cell. In the event of this occurance, there are heaters that are attached to the cell which will be turned on at a certain temperature, as determined by the operator. The heaters are controlled by an `Omron temperature Controller <http://products.omron.us/item/in-panel-controllers/e5dc/e5dc-qx2dsm-002>`__
+Because of the heat generated by such a lamp, before sending the light into the monochromator, the light travels through a water cell which absorbs much of the infrared radiation. This water heats up, so the white light source must be operated with a chiller, discussed below. The chiller should keep the water cool enough, but if there is ever a situation where there is too much heat being sent into the monochromator, there is a thermal fuse attached to the entrance to the monochromator. When it hits a certain temperature it will trigger the emergency stop. The WLS will also experience freezing temperatures on the summit. To avoid freezing the water in the cell, there are heaters attached to the cell which will be turned on at a certain temperature, as determined by the operator. The heaters are controlled by an `Omron temperature Controller <http://products.omron.us/item/in-panel-controllers/e5dc/e5dc-qx2dsm-002>`__
```

Comment:
....but if there is ever a situation where there.....
the most likely event of overheating is if the chiller fails, so maybe add that as the potential failure scenario.
I think you can also start a new paragraph at "The WLS will also experience freezing..."

Reply:
done